schema/release-schema.json

```diff
@@ -91,11 +91,13 @@
     },
     "language": {
       "title": "Release language",
-      "description": "The default language of the data using either two-letter [ISO639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), or extended [BCP47 language tags](https://www.w3.org/International/articles/language-tags/). The use of lowercase two-letter codes from [ISO639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) is recommended.",
+      "description": "The default language of the data using either two-letter codes from the from the open [ISO639-1](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#iso639-1) codelist, or extended [BCP47 language tags](https://www.w3.org/International/articles/language-tags/). The use of lowercase two-letter codes from the ISO639-1 codelist is recommended.",
```

Comment:
"from the" repetition.

Let's align both fields:

`... [semantics] ... , from the open [language](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#language) codelist. A [BCP47 language tag](https://www.w3.org/International/articles/language-tags/) is allowed, if there is a user need for the additional information.`

Reply:
Done in https://github.com/open-contracting/standard/pull/1316/commits/bb7a0c653f35f8aa6b39c4531f4b218214fbc02d